### DISCUSSION TITLE
how to move a pane to another tab?

### DISCUSSION THREAD

**Discussion by Freed-Wu** [zellij-org/zellij#3606] [Q&A] [answered]
For tmux, `tmux move-pane -t 3:1` can move current pane to tab 3's 1st pane.

How can zellij do it?

**Comment by Lockszmith-GH** [comment]
Welcome @Freed-Wu, I see you've started using Zellij (with your postings comparing tmux), and you've probably already read other responses from me (just another user like you, not officially associated, I'm not a contributor).

This isn't implemented yet, a couple of weeks ago a [PR was pushed](https://github.com/zellij-org/zellij/pull/3576) with API updates that might allow this shortly.

You can track this in the [CHANGELOG](https://github.com/zellij-org/zellij/blob/main/CHANGELOG.md?plain=1) (search for 3576)

**Comment by imsnif** [comment, marked as answer]
This actually is possible. By default: `Ctrl t` + `b` (new tab), `Ctrl t` + `[]` left/right

**Reply by valterschutz** [reply]
This works great but I struggled to find this by myself since the `BreakPane` option is not listed under the "Possible Actions" [documentation](https://zellij.dev/documentation/keybindings-possible-actions.html).
